hollow/src/main/java/com/netflix/hollow/api/producer/validation/ValidatorStatus.java

```diff
@@ -0,0 +1,147 @@
+/*
+ *
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.hollow.api.producer.validation;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.Generated;
+
+import com.netflix.hollow.api.producer.HollowProducerListener.Status;
+/**
+ * *************************************
+ * NOTE: Beta API subject to change.   *
+ * *************************************
+ * 
+ * @author lkanchanapalli {@literal<lavanya65@yahoo.com>}
+ * 
+ * ValidationStatus has one instance per each validator that ran. 
+ * For now ValidationStatus builds these and sets toString on validator as the message (where validators provide any details).
+ * In next iteration this might be directly returned by validators.
+ */
+class ValidatorStatus {
+    private final long version;
+    private final Status status;
+    private final String message;
+    private final Throwable throwable;
+    private final Map<String, String> additionalInfo;
+
+	@Generated("SparkTools")
```

Comment:
It is just indicating a plugin I used to generate the code. As far as I know this annotation does not impact the behavior. I will add comment explaining or remove it.

Reply:
We should probably not check this in.

Reply:
I checked the @Generate annotation. The documentation clearly states that it is used to indicate generated code  and a tool name can be provided. With that documentation I do not feel this annotation is an issue.

Reply:
It would be great if the plugin has option to omit this ;)  Similar to Eclipse building code generator toString(), equals() which does not add annotation which keeps things cleaner.

Reply:
Hmm. But the @generate seems pretty common annotation used in Java specifically for this. Since I don't feel too strongly about this, I will remove it based on more than one person finding this confusing.

Reply:
Done. Removed the annotations.